.github/workflows/unit-test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Run Tests
         run: |
           cd ${{ github.workspace }}/main/
-          xcodebuild -scheme Unit\ Tests -workspace PortalSwift.xcworkspace test -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 14 Pro,OS=16.2'
+          xcodebuild -quiet -scheme Unit\ Tests -workspace PortalSwift.xcworkspace test -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 14 Pro,OS=16.2'
```

Comment:
does this added flag hide the output?

Reply:
It limits the output to warnings and errors, so we can more easily diagnose failed unit tests in CI. I ended up needing this to get tests passing in CI, so I just left it in 🙂